cmd/cdi-controller/controller.go

```diff
@@ -75,6 +66,14 @@ func init() {
 	glog.V(Vdebug).Infof("init: complete: cdi controller will create importer using image %q\n", importerImage)
 }
 
+func getRequiredEnvVar(name string) string {
+	val := os.Getenv(name)
+	if val == "" {
+		glog.Fatalf("Environment Variable %q undefined\n", name)
```

Comment:
`*.Fatal` in a module is usually a pretty bad deal from a debug view point.  Could we move this into the main caller so it's easier to know what's going on if things go south?